_docs/principles/accessibility.md

```diff
@@ -121,27 +216,27 @@ Other optional indicators to consider are the following elements:
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
 
-### Color and contrast
+## Page layout guidelines
 
-Readability requires sufficient contrast between the text and its background. Low contrast is especially problematic for users with low vision who also often have contrast sensitivity.
+Consider the following page layout guidelines when you construct an accessible UI.
 
-To ensure text is readable, Helix adheres to the accessibility standards defined by [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/TR/WCAG21/). These standards are based on contrast ratios calculated from text size, text color, and background color.
+{% endcolumn %}
 
-Helix minimum contrast standards require the following elements:
+</div>
 
-- Small text should have a contrast ratio of at least 4.5:1 against its background color.
-- Large text (at 14px bold or 18px regular and up) should have a contrast ratio of at least 3:1.
-- We recommend using the contrast ratios for icons or other critical elements.
+</section>
 
-{% endcolumn %}
+<section class="static-section"  markdown="1">
 
-{% column right:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+<div class="hxRow"  markdown="1">
 
-{% figure [caption: "Helix typography styles with their corresponding contrast ratios. All ratios were calculated using a `#fafafa` (`gray 25`) background color."] [class:"image bg-light border"] %}
-<embed src="{{site.url}}/assets/images/overview/accessibility/accessibility-typography.png" width="598"/>
-{% endfigure %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+### Be consistent with headings
+
+Some screen readers allow users to listen to all of the headings on a page and then choose which content area they want to explore. Because screen readers read header tags in the order they appear, it is important that you not skip header levels. The structure of a page hierarchy should be logical. For example, don't skip from a Header 1 to a Header 3, and then back to a Header 2.
```

Comment:
Ok, this is definitely nuance I didn't pick up in my research. How about this for a description?

"Screen readers read headings in the order you define them in the document, and not in the order they appear on the screen. For example, a Heading 1 and a Heading 2 can appear in that order on the screen, but you can define them in the document so that Heading 2 is read before Heading 1. 

Ensure that you are consistent with the order the headings appear on the screen and the order you define the headings in the document."